ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -1128,6 +1128,13 @@ IE_CPU.onnx_resize11_scales_nearest_asymmetric_floor_dynamic_sizes
 # Input data precision not supported. Expected float.
 ctc_greedy_decoder_f16
 
+# Wrong output when axis 0
+IE_CPU.log_softmax_1d_single_value
+IE_CPU.log_softmax_2d_axis0
+IE_CPU.log_softmax_2d_axis_neg2
+IE_CPU.log_softmax_3d_axis_0
+IE_CPU.log_softmax_3d_axis_neg3
```

Comment:
Could you please explain what do these line mean? And why aren't there GPU backend here?

Reply:
These lines mean excluded tests, CPU plugin outputs the wrong result for axis 0. Gpu plugin works, as tests show, but GPU plugin still complains that LogSoftmax is not supported, so I am not enabling it for GPU in this PR.

Reply:
I am lost a bit:) As I understand from your comment you also need to exclude running tests on GPU and add lines IE_GPU.*** here. Could you please explain?

Reply:
I think I can explain how those files work - we use them to skip tests in the CI so that they don't block PRs. In this case those tests are skipped only for the CPU plugin and not for the GPU. Since CPU is the only one tested in CI, the tests are not executed on GPU anyway. If GPU testing gets added - we'll probably have to review the whole file and exclude many more.

The way to exclude the test for both CPU and GPU is to add a line that contains the test name only - withouth the `IE_CPU.` or `IE_GPU.` prefix.

Reply:
@tomdol, thank you very much for the explanation. Maxim, you seem to need a list of tests without plugin name prefix.
@tomdol, do you know if we can switch on/off nGraph transformation manually in backend tests in order to test computational results - fusion and decomposition for `LogSoftmax`, for example?

Reply:
@ilyachur, @GlebKazantaev, are aware of if we can switch on/off nGraph transformation manually in backend tests in order to test inference results after transformations

Reply:
Locally I think this is possible. When you have an ngraph::Function and create a CNNNetwork out of it, those transformations are executed on the way. A big bunch of them belongs to something called common optimizations https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp but there's probably more that's happening somewhere else. I could not find the fusion nor decomposition pass for this op. Is this already in master?

Reply:
The fusion and decomposition transformations are in this PR. Please see log_softmax_fusion.cpp file

Reply:
@tomdol @rkazants Not, test without prefix will disable all tests, INTERPRETER too. I ran tests locally and GPU tests were successful for me, and if I alter expected results they fail, so GPU tests works, which means no need to disable them.

Reply:
@mvafin, do we have implemented `LogSoftMax` on GPU or is it decompossed to supported ops?

Reply:
@mvafin the INTERPRETER backend is handled in a separate file https://github.com/openvinotoolkit/openvino/blob/master/ngraph/test/runtime/interpreter/unit_test.manifest, the one you modified only works for IE_CPU and IE_GPU plugins.